miniwin/include/miniwin/miniwindevice.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+DEFINE_GUID(IID_IDirect3DRMMiniwinDevice, 0x6eb09673, 0x8d30, 0x4d8a, 0x8d, 0x81, 0x34, 0xea, 0x69, 0x30, 0x12, 0x01);
+
+struct IDirect3DRMMiniwinDevice : virtual public IUnknown {
+	virtual float GetShininessFactor() = 0;
+	virtual HRESULT SetShininessFactor(float factor) = 0;
+	virtual HRESULT GetDescription(char* buffer, int bufferSize) = 0;
```

Comment:
A more viable option would be to use the names gotten from `EnumDevicesCallback()` that way it works with DirectX as well..